<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpacityEditor</class>
 <widget class="QDialog" name="OpacityEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Opacity Map Editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Opacity:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGlue::OpacityMapPalette" name="qglOpacityMapPalette">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="page_0">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>322</width>
        <height>285</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Freeform Curve Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QPushButton" name="resetButton">
           <property name="text">
            <string>Reset</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="undoButton">
           <property name="text">
            <string>Undo</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="redoButton">
           <property name="text">
            <string>Redo</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>322</width>
        <height>285</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Opacity Expression Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,4">
       <item>
        <layout class="QHBoxLayout" name="expression_o_box" stretch="0,0">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>O:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="expression_o">
           <property name="text">
            <string>x*0.8</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>322</width>
        <height>285</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Control Point Editor</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="controlPointTable">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAsNeeded</enum>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustIgnored</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>65</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderDefaultSectionSize">
          <number>24</number>
         </attribute>
         <attribute name="verticalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>1</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>2</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>3</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>4</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>5</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>6</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>7</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>8</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>9</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <row>
          <property name="text">
           <string>10</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Point</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Opacity</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
            <underline>true</underline>
           </font>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>0.0</string>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string>0.0</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>0.5</string>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string>0.0</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>0.75</string>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string>0.4</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>1.0</string>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string>0.4</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QGlue::OpacityMapPalette</class>
   <extends>QOpenGLWidget</extends>
   <header>QGlue/opacitymappalette.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
